src/api_server/src/crypto.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::rand::Rng;
+use aes_gcm::{
+    aead::{Aead, KeyInit, OsRng},
+    Aes256Gcm, Nonce,
+};
+use anyhow::{anyhow, Result};
+use kbs_types::{Response, TeePubKey};
+use rsa::pkcs8::DecodePublicKey;
+use rsa::{PaddingScheme, PublicKey, RsaPublicKey};
+use serde_json::json;
+
+const RSA_KEY_TYPE: &str = "RSA1_5";
```

Comment:
According to the [JWA RFC](https://www.rfc-editor.org/rfc/rfc7518.html#page-28), `kty` should be `RSA`.